docs/guide/http_cache.md

```diff
@@ -524,7 +524,7 @@ eZ Platform already interferes with the hash generation process by adding the cu
 
 ##### New anonymous `X-User-Hash`
```

Comment:
For 2.x we can even remove this section afaik. @kmadejski right?

Reply:
I talked with @DominikaK about this yesterday, personally, I think that we can keep these instructions for getting `X-User-Hash` manually, just in case if somebody needs it for some debugging. Maybe this part can be stated differently, because for now it sounds like something necessary in case of changed user / group / permission. Besides that, totally right, this is not required to configure Varnish with 1.13 and up (with `PURGE` and `xkey`).